docs/src/components/icons-list.astro

```diff
@@ -37,4 +62,18 @@ const icons = Object.keys(Icons) as (keyof typeof Icons)[];
 		margin: 0;
 		padding: 0.75rem;
 	}
+
+	.icon-preview:hover {
+		cursor: pointer;
+		border: 1px solid var(--sl-color-gray-4);
+		background: var(--sl-color-gray-5);
+	}
+
+	.copied {
+		display: none;
+	}
+
+	.icon-name {
+		display: none;
+	}
```

Comment:
Now that the element is focusable through keyboard, we can add the same effect from when it is hovered over. And with the code changes made, we can also remove the `.copied` and `.icon-name` classes.
```suggestion
	.icon-preview:hover, .icon-preview:focus {
		cursor: pointer;
		border: 1px solid var(--sl-color-gray-4);
		background: var(--sl-color-gray-5);
	}
```